blottertrax/config.py

```diff
@@ -18,13 +18,13 @@ class Config:
     SOUNDCLOUD_KEY: str = ''
 
     def __init__(self):
-        config = configparser.ConfigParser()
+        config = configparser.ConfigParser(strict=False, interpolation=None)
 
         try:
-            config.read('{}/conf/config.ini'.format(os.path.dirname(os.path.realpath(__file__))))
+            config.read('{}/../conf/config.ini'.format(os.path.dirname(os.path.realpath(__file__))))
             self.CLIENT_ID = config['REDDIT']['CLIENT_ID']
             self.CLIENT_SECRET = config['REDDIT']['CLIENT_SECRET']
-            self.PASSWORD = config['REDDIT']['PASSWORD']
+            self.PASSWORD = config.get('REDDIT', 'PASSWORD')
```

Comment:
Any reason not to use this same syntax everywhere?  I assume this is to get around the need to escape special characters in the password?

Reply:
it's not actually, just trying out if it makes a difference and it didn't. I will make them all getters though

Reply:
Honestly either syntax is fine by me if it doesn't fix any specific issue.  Just think they should all be the same if possible :)

Reply:
Actually, https://stackoverflow.com/questions/47640354/reading-special-characters-text-from-ini-file-in-python seems to say we could use RawConfigParser instead and get around the % interpolation issue in the password field.